### PR TITLE
Add mod smaller on button

### PR DESCRIPTION
**Pull request by almonais**
image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1035

File: packages/scss/src/theming/components/_button.theme.scss [hunk 1]
```diff
@@ -10,5 +10,10 @@ $button: (
 		line-height: _theme("sizes.small.line-height"),
 		padding: _theme("spacings.smaller") _theme("spacings.small"),
 	),
+	smaller: (
+		font-size: _theme("sizes.smaller.font-size"),
+		line-height: _theme("sizes.smaller.line-height"),
+		padding: _theme("spacings.smaller") _theme("spacings.smaller"),
```
Comment (sanlucca) [inline packages/scss/src/theming/components/_button.theme.scss:16]: it should be `_theme("spacings.smallest") _theme("spacings.smaller")`

File: packages/scss/src/components/_button.scss [hunk 2]
```diff
@@ -98,6 +98,17 @@
 		}
 	}
 
+	&.mod-smaller {
+		font-size: _component("button.smaller.font-size");
+		line-height: _component("button.smaller.line-height");
+		padding: _component("button.smaller.padding");
+
+		.lucca-icon {
+			line-height: 1rem;
```
Comment (sanlucca) [inline packages/scss/src/components/_button.scss:107]: You should use sizes from the theming here

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1035

**Pull request merged**